### PR TITLE
zookeeper 3.5.7

### PR DESCRIPTION
**Pull request by EricFromCanada**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by EricFromCanada** [comment]
For the client, autoconf would always complain about the absence of `cppunit` even if it was made available. Switching to Cmake worked, although installation needs to be done manually and it only builds static libs.

File: Formula/zookeeper.rb [hunk 1]
```diff
@@ -12,15 +13,8 @@ class Zookeeper < Formula
...
-    url "https://svn.apache.org/repos/asf/zookeeper/trunk"
-
-    depends_on "ant" => :build
-    depends_on "autoconf" => :build
-    depends_on "automake" => :build
-    depends_on "cppunit" => :build
-    depends_on "libtool" => :build
-  end
+  depends_on "ant" => :build
+  depends_on "cmake" => :build
```
Comment (chenrui333) [inline Formula/zookeeper.rb]: 👍

**Comment by Bo98** [comment]
The shared libraries being removed is somewhat concerning, considering we don't know if anyone was using it. That said, we've managed to get away with stuff like that before.

Just to say though: the `cppunit` error looks like it could have been fixed by depending on `pkg-config`.

**Comment by EricFromCanada** [comment]
Yep, that's fixed it. Switched back to autoconf for now.

**Pull request closed without merging**